pkg/network/ebpf/c/prebuilt/tracer.c

```diff
@@ -820,6 +850,74 @@ int kprobe__ip_make_skb(struct pt_regs* ctx) {
     return 0;
 }
 
+#if ROUTE_LOOKUP_USE_EBPF
```

Comment:
This should live in a different file (not under the prebuild directory) once the initial runtime compilation  PR is merged.